iphone/Maps/Classes/MapsAppDelegate.mm

```diff
@@ -222,12 +226,19 @@ - (void)applicationDidBecomeActive:(UIApplication *)application {
     f.OnRecoverSurface(static_cast<int>(objcSize.width), static_cast<int>(objcSize.height),
                        true /* recreateContextDependentResources */);
   }
+  [self startCheckingClipboard];
   [MWMLocationManager applicationDidBecomeActive];
   [MWMSearch addCategoriesToSpotlight];
   [MWMKeyboard applicationDidBecomeActive];
   [MWMTextToSpeech applicationDidBecomeActive];
   LOG(LINFO, ("applicationDidBecomeActive - end"));
 }
+- (void)startCheckingClipboard {
+    NSLog(@"Watching started");
+  ClipboardChecker *clipboardChecker = [[ClipboardChecker alloc] init];
+  UIWindow *targetWindow = [UIApplication sharedApplication].keyWindow;
+  [clipboardChecker startChecking:targetWindow];
```

Comment:
If the code runs in one function, can we avoid creating class at all, and define one function that internally may call a few other private functions?